app/controllers/state_file/questions/nj_retirement_warning_controller.rb

```diff
@@ -7,7 +7,11 @@ def self.show?(intake)
         line_15 = intake.calculator.line_or_zero(:NJ1040_LINE_15)
         line_16a = intake.calculator.line_or_zero(:NJ1040_LINE_16A)
         retirement_helper = Efile::Nj::NjRetirementIncomeHelper.new(intake)
-        retirement_helper.show_retirement_income_warning?(line_15, line_16a) && Flipper.enabled?(:show_retirement_ui)
+        should_show = retirement_helper.show_retirement_income_warning?(line_15, line_16a) && Flipper.enabled?(:show_retirement_ui)
+        if should_show && intake.eligibility_retirement_warning_continue_unfilled?
+          intake.eligibility_retirement_warning_continue = :shown
```

Comment:
i don't think this works. i tested it locally and the value goes back to `unfilled` when i put a break point in `edit`